spec/support/shared_examples/http_request_methods.rb

```diff
@@ -0,0 +1,29 @@
+shared_examples 'available http request methods' do
+  it "instantiates a new Request object" do
+    VCR.use_cassette("client/#{http_method}/request") do
+      expect(Myfinance::Request).to receive(:new).and_call_original
+      subject.send(http_method, url, body: params)
+    end
+  end
+
+  it 'runs the request' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.